src/main/java/com/zendesk/maxwell/producer/HttpPostProducer.java

```diff
@@ -0,0 +1,115 @@
+package com.zendesk.maxwell.producer;
+
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpResponseException;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.HttpHeaders;
+import com.google.api.client.http.ByteArrayContent;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpBackOffUnsuccessfulResponseHandler;
+import com.google.api.client.util.ExponentialBackOff;
+
+import com.google.api.client.http.javanet.NetHttpTransport;
+import org.apache.commons.codec.binary.Hex;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import java.io.IOException;
+
+public class HttpPostProducer extends AbstractProducer {
+    static final String HTTP_HMAC_HEADER = "x-signature";
+    static final String HTTP_CONENT_TYPE_HEADER = "application/json";
+    static final String HMAC_ALG = "HmacSHA1";
+
+    private static final int maxRetryTimeSeconds = 15;
+    private final GenericUrl httpPostEndPoint;
+    private final String httpPostHmacSecret;
+    private final HttpTransport httpTransport = new NetHttpTransport();
+
+	public HttpPostProducer(MaxwellContext context, String httpPostEndPoint, String httpPostHmacSecret) {
+		super(context);
+        this.httpPostEndPoint = new GenericUrl(httpPostEndPoint);
+        this.httpPostHmacSecret = httpPostHmacSecret;
+	}
+
+	@Override
+	public void push(RowMap r) throws Exception {
+
+        Set<String> eventTypeWhiteList = new HashSet<String>(Arrays.asList("insert", "update", "delete"));
+
+        String rowType = r.getRowType();
+        if ( eventTypeWhiteList.contains(rowType) ) {
```

Comment:
this seems odd; what event types are you avoiding here?  #505 will add `RowMap#shouldOutput(outputConfig)`, which might get you what you want.